source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB specific information to users,
```

Comment:
[Suggestion]

I think "MongoDB specific" should be hyphenated:

```suggestion
The ``/docs`` command provides MongoDB-specific information to users,
```

Reply:
Done!